test/seed/03-patients.ts

```diff
@@ -0,0 +1,56 @@
+import clean from '../helper/clean';
+
+import { mails } from './02-psychologists';
+
+import uuid from '../../utils/uuid';
+import { patientsTable } from '../../db/tables';
+import { Knex } from 'knex';
+import { Psychologist } from '../../types/Psychologist';
+
+let patientsByPsychologist = {};
+export const getPatientsByPsychologist = (): {
+  [key: string]: Psychologist
+} => patientsByPsychologist;
+
+export const seed = async (knex: Knex, fixValue = false): Promise<void> => {
```

Comment:
I would rename `fixValue = false` by `randomValues = true` for better readability.
When I first read "fixValue" I though it was "fix" as in "bugfix".

Reply:
i wanted to do that, but knex call the function with no second params (so false) so I can't change the logic and need a negative name, I'm open to suggestion ?

Reply:
`fixedValue`?